app/models/ldap_identity_provider.rb

```diff
@@ -56,8 +58,12 @@ def ldap_search(filter:)
       size: 500,
```

Comment:
ahh, is this same 500? we should maybe put this in config.

Reply:
:man_shrugging: